apps-rendering/src/components/RichLink/index.tsx

```diff
@@ -56,6 +56,13 @@ const liveBlogRichLinkStyles = css`
 	}
 `;
 
+const analysisRichLinkStyles = css`
+	background-color: #f2e8e6;
+	:hover {
+		background-color: #e9d9d5;
+	}
+`;
```

Comment:
It seems this colour doesn't exist in source palette, but I was wondering if it's worth adding it there? What do you think?

Reply:
Nice one Sophie 👍